tee-worker/litentry/core/vc-task/receiver/src/lib.rs

```diff
@@ -82,8 +82,7 @@ pub fn run_vc_handler_runner<ShieldingKeyRepository, A, S, H, O, Z, N>(
 	N::MetadataType: NodeMetadataTrait,
 {
 	let vc_task_receiver = init_vc_task_sender_storage();
-	let n_workers = 4;
-	let pool = ThreadPool::new(n_workers);
+	let pool = ThreadPool::new().unwrap();
```

Comment:
It uses the default builder config - did you check what the pool size it? (It's bound to number of cpus but I don't know the number actually)

Reply:
Yeah I'm using the default, It is equal to the number of cpus available on the machine. I think i mentioned it in the issue.

Reply:
I just want to make sure it's not some value like 1 :D as enclave is quite a special environment, I won't be surprised if the sdk (Hw or SW) is doing some schenanigans

Reply:
I don't think to give default is a good idea. Quote "It is equal to the number of cpus available on the machine.", "available" means at that moment of executing or in general the CPU physically has? what `num_cpus::get()` gives out?

Reply:
And I think it will also change the behaviour of batch_VC PR. Please hold this PR for a moment.